src/Services/Basket/Basket.API/Model/CustomerBasket.cs

```diff
@@ -1,21 +1,18 @@
-﻿using System.Collections.Generic;
-
-namespace Microsoft.eShopOnContainers.Services.Basket.API.Model
+﻿namespace Microsoft.eShopOnContainers.Services.Basket.API.Model;
+public class CustomerBasket
 {
-    public class CustomerBasket
-    {
-        public string BuyerId { get; set; }
+    public string BuyerId { get; set; }
 
-        public List<BasketItem> Items { get; set; } = new List<BasketItem>();
+    public List<BasketItem> Items { get; set; } = new List<BasketItem>();
```

Comment:
```suggestion
    public List<BasketItem> Items { get; set; } = new();
```

For property initializers, you can use the implicit target type new - since the type is obvious.